public/style.css

```diff
@@ -1,4 +1,121 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+
+body {
+  background: #282c34;
+  color: #fff;
+  font-family: 'Open Sans', sans-serif;
+  padding: 30px;
+  text-align: center;
+}
+
 h1 {
   font-weight: lighter;
-  font-style: italic;
+}
+
+p {
+  color: #999;
+  font-size: 0.8rem;
+}
+
+a {
+  color: #999;
+  font-size: 0.8rem;
+}
+
+label {
+  display: block;
+  margin: 1rem;
+}
+
+input,
+textarea {
+  background: rgba(0,0,0,0.2);
+  padding: 0.6rem 1rem;
+  border: none;
+  margin: 0 1rem;
+  border-radius: 0.5rem;
+  outline: none;
+  color: #fff;
+  font-size: inherit;
+}
+
+textarea {
+  margin: 1rem auto;
+  width: 400px;
+  height: 200px;
+  display: block;
+}
+
+button {
+  background: #0064e7;
+  border: none;
+  border-radius: 15px;
+  padding: 10px 25px;
+  color: #fff;
+  font-family: inherit;
+  margin: 20px 10px;
+  text-transform: uppercase;
+  outline: none;
+  transition: .2s;
+  cursor: pointer;
+}
+
+.isError {
+  background: #fff;
+  color: #000;
+  border-left: 4px solid #111;
+  padding: 0.9rem 0.8rem;
+  margin: 1rem auto 1.5rem auto;
+  display: flex;
+  opacity: 1;
+  font-weight: lighter;
+  box-shadow: 0 0 0 rgba(0,0,0,0.2);
+  max-width: 15rem;
+  border-radius: 4px;
+  background: #ff5d6c;
+  color: #fff;
+  border-color:#ea2027;
+  -webkit-animation-name: fadeIn;
+  animation-name: fadeIn;
+  animation-duration: 1s;
+}
+
+.isSent {
```

Comment:
style w tej klasie się powtarzają

Reply:
Trochę nie rozumiem co mam zrobić - owszem większość styli się powtarza, ale są inne kolory czcionek. Mam wydzielić te same style i zamiast jednej klasy dać dwie w divie? To na pewno dobra praktyka, żeby zwiększać liczbę klas?
PS. Ten kawałek kodu jest skopiowany prosto z Kodilli

Reply:
chodzi o to, że powtarzają się style w jednej klasie. Masz 

```
background: #ff5d6c;
  color: #fff;
  ```
  
  zdefiniowane 2 razy w tej samej klasie

Reply:
chodzi o to, żeby usunąć te, które się powtarzają

Reply:
Faktycznie, moj błąd